datadog_checks_base/tests/base/utils/db/test_util.py

```diff
@@ -254,3 +258,21 @@ def test_dbm_async_job_inactive_stop(aggregator):
     job.run_job_loop([])
     job._job_loop_future.result()
     aggregator.assert_metric("dd.test-dbms.async_job.inactive_stop", tags=['job:test-job'])
+
+
+@pytest.mark.parametrize(
+    "input",
+    [
+        {"foo": "bar"},  # dict
+        {"foo": "bar", "baz": 1},  # dict with multiple keys
+        {"foo": "bar", "baz": 1, "qux": {"quux": "corge"}},  # nested dict
+        {"foo": b'bar'},  # dict with bytes
+        {"foo": decimal.Decimal("1.0")},  # dict with decimal.Decimal
+        {"foo": datetime.datetime(2020, 1, 1, 0, 0, 0)},  # dict with datetime.datetime
+        {"foo": datetime.date(2020, 1, 1)},  # dict with datetime.date
+        {"foo": IPv4Address(u"192.168.1.1")},  # dict with IPv4Address
```

Comment:
nit: instead of adding a comment, you can in fact name the test! For example: https://github.com/DataDog/integrations-core/blob/0ce4ba15d52759e13bc97d53e42470f082c689ee/mongo/tests/test_mongo.py#L55-L63

Thanks to that pytest will also display this name in the terminal when running them. In this case: 

```
tests/test_mongo.py::test_mongo_authdb[standard] PASSED                  [ 42%]
tests/test_mongo.py::test_mongo_authdb[standard-alternative] PASSED      [ 43%]
tests/test_mongo.py::test_mongo_authdb[legacy] PASSED                    [ 44%]
```

Your version: 

```
tests/base/utils/db/test_util.py::test_default_json_event_encoding[input0] PASSED [ 76%]
tests/base/utils/db/test_util.py::test_default_json_event_encoding[input1] PASSED [ 77%]
tests/base/utils/db/test_util.py::test_default_json_event_encoding[input2] PASSED [ 77%]
tests/base/utils/db/test_util.py::test_default_json_event_encoding[input3] PASSED [ 77%]
tests/base/utils/db/test_util.py::test_default_json_event_encoding[input4] PASSED [ 77%]
tests/base/utils/db/test_util.py::test_default_json_event_encoding[input5] PASSED [ 77%]
tests/base/utils/db/test_util.py::test_default_json_event_encoding[input6] PASSED [ 77%]
tests/base/utils/db/test_util.py::test_default_json_event_encoding[input7] PASSED [ 77%]
```